app.py

```diff
@@ -45,6 +46,18 @@
 _LOGGER = logging.getLogger("thoth.revsolver")
 _QUERY_COUNT = int(os.getenv("THOTH_REVSOLVER_QUERY_COUNT", GraphDatabase.DEFAULT_COUNT))
 
+# metrics
+THOTH_METRICS_PUSHGATEWAY_URL = os.getenv("PROMETHEUS_PUSHGATEWAY_URL")
```

Comment:
this could be None, right?

Reply:
Yes it can, but there is a check below if it is missing, not to break the actual logic of the application. Metrics are sent if the env variables are provided, otherwise not, but the application keeps doing the job, should we these metrics mandatory as well?